packages/data-stores/src/site/types.ts

```diff
@@ -77,17 +77,112 @@ export interface SiteDetailsPlan {
 	is_free: boolean;
 }
 
+export interface DifmLiteSiteOptions {
+	site_category?: string;
+	is_website_content_submitted?: boolean;
+	selected_page_titles: string[];
+}
+
 export interface SiteDetails {
 	ID: number;
 	name: string | undefined;
 	description: string;
 	URL: string;
 	launch_status: string;
 	options: {
-		created_at: string;
+		admin_url?: string;
+		advanced_seo_front_page_description?: string;
+		advanced_seo_title_formats?: string[];
+		ak_vp_bundle_enabled?: boolean;
+		allowed_file_types?: string[];
+		anchor_podcast?: boolean;
+		background_color?: boolean;
+		blog_public?: number;
+		created_at: Date;
```

Comment:
🤔  I'm curious why this property was changed to `Date` from `string`? Looking at it in Redux, it appears to be a string like `"2005-10-30T04:28:50+00:00"`.

I'm assuming this was probably a mistake so I'm changing it back to `string` in https://github.com/Automattic/wp-calypso/pull/66948 but let me know if there's something important I'm missing!